src/admin-views/tribe-options-display.php

```diff
@@ -134,10 +134,10 @@
 				tribe_get_ticket_label_plural( 'tickets_use_new_views' )
 			)
 		),
-		'tooltip'         => esc_html(
+		'tooltip'         => wp_kses_post(
 			sprintf(
 			// Translators: %1$s: dynamic "Tickets" text.
-				_x( 'This setting will enable the updated front-end views and Individual Attendee Collection (IAC) flows for %1$s.', 'settings tooltip', 'event-tickets' ),
+				_x( 'Enables updated front-end views and Individual Attendee Collection (IAC) flows for %1$s. You\'ll see a new default setting within the Tickets tab and new individual ticket setting within both Classic and Gutenberg editors <a href="https://m.tri.be/kb">[Learn more]</a>.', 'settings tooltip', 'event-tickets' ),
 				tribe_get_ticket_label_plural( 'tickets_use_new_views' )
 			)
```

Comment:
move the anchor tags out into the sprintf so they are not required to be translated, and also allows moving around of the anchor tags in languages where the grammar doesn't work the same as English, something like:

```
sprintf(
// Translators: %1$s: dynamic "Tickets" text.
	_x( 'Enables updated front-end views and Individual Attendee Collection (IAC) flows for %1$s. You\'ll see a new default setting within the Tickets tab and new individual ticket setting within both Classic and Gutenberg editors %2$s[Learn more]%3$s.', 'settings tooltip', 'event-tickets' ),
	tribe_get_ticket_label_plural( 'tickets_use_new_views' ),
	<a href="https://m.tri.be/kb">,
	</a>
)
```